package.json

```diff
@@ -34,7 +34,9 @@
     "watch-live-reload": "./node_modules/.bin/grunt watch",
     "lint": "./node_modules/.bin/standard --fix",
     "lint-sass": "./node_modules/.bin/sass-lint -v",
-    "test": "npm run snyk-protect && rm -rf ./pacts && NODE_ENV=test ./node_modules/mocha/bin/mocha --exclude **/*.cy.test.js '!(node_modules)/**/*.test'.js",
+    "test": "npm run snyk-protect && npm run test:clean-pacts-run-mocha",
```

Comment:
Spoke with @sfount and @stephencdaly - will only remove `snyk-protect` from `npm test`.
I will look into whether we can completely remove it from the Self service repo because it is handled on Self service.